tests/src/testcommon.hh

```diff
@@ -92,6 +93,7 @@ struct CornerFactory {
 };
 
 enum class CornerDistortionFlag { unDistorted, fixedDistorted, randomlyDistorted };
+enum class ElementGeometryFlag { simplex, cube };
```

Comment:
Overall IMHO this should not really exist but It should be done with the Dune internals for cube and simplex. Even though the functionality here is for more dimension, i think it would be nicer, if the tests would work with cube(dim) or simplex(dim) and then internally the reference element can be asked for quantities.